src/controllers/hid/hidcontroller.h

```diff
@@ -72,7 +72,7 @@ class HidController final : public Controller {
     static constexpr int kNumBuffers = 2;
     static constexpr int kBufferSize = 255;
     unsigned char m_pPollData[kNumBuffers][kBufferSize];
-    int m_LastPollSize;
+    int m_lastPollSize;
```

Comment:
I did not intend for this to be applied literally from GitHub. You need to change the variable name in the rest of the code.

Reply:
Will do!